Manager/FormManager.php

```diff
@@ -82,12 +82,12 @@ public function save($data, $locale = null, $id = null)
     {
         $form = new Form();
 
-        // Find exist or create new entity
+        // Find exist or create new entity.
```

Comment:
this would be a "sentence" and the point would be ok (: but you can leave it as it is!